examples/src/nullable_field/main.rs

```diff
@@ -0,0 +1,17 @@
+use proto_nullable_field::basic::{
+    Inner,
+    NonNullableMessage,
+};
+
+fn main() {
+    let _msg1 = NonNullableMessage {
```

Comment:
Can you leave a comment here explaining how you used the rust.nullable_field option? Just something to link it back to the proto

Reply:
Updated! Added comments and actually some asserts to show the difference in behavior (to help w/ questions like in #125)